larray/core/checked.py

```diff
@@ -383,13 +383,15 @@ class CheckedSession(Session, AbstractCheckedSession, metaclass=ModelMetaclass):
         dtype: int32
         memory used: 968 bytes
 
-        >>> # note that it still possible to add undeclared variables to a checked session
-        >>> # but this must be done with caution.
-        >>> m.undeclared_var = 'undeclared_var'
+        It is possible to add undeclared variables to a checked session
+        but this will print a warning:
+
+        >>> m.undeclared_var = 'my_value'                   # doctest: +SKIP
```

Comment:
Is the warning tested in test_checkedsession.py ?

Reply:
yes (cs.ax = 'ax')